torch/csrc/jit/codegen/cuda/lower_expr_sort.cpp

```diff
@@ -333,30 +339,30 @@ class ExprSegmentationSorter {
 };
 
 // // Debug printing, disabled due to clang-tidy see above for declarations.
-// std::ostream& operator<<(std::ostream& os, ExprGroup* group) {
-//   os << "Group Start{\n  ca, pa ("
-//      << group->payload()->ca_domains_.size() << ", "
-//      << group->payload()->pa_domains_.size() << ")";
-//   os << " ca_ids {";
-//   for (size_t i = 0; i < group->payload()->ca_domains_.size(); i++) {
-//     os << group->payload()->ca_domains_[i];
-//     if (i + 1 != group->payload()->ca_domains_.size())
-//       os << ", ";
-//   }
-//   os << "} pa_ids {";
-//   for (size_t i = 0; i < group->payload()->pa_domains_.size(); i++) {
-//     os << group->payload()->pa_domains_[i];
-//     if (i + 1 != group->payload()->pa_domains_.size())
-//       os << ", ";
-//   }
-//   os << "}";
-//   os << "\nExprs {\n";
-//   for(auto expr : group->exprs()){
-//     os << expr;
-//   }
-//    os << "}Group End\n";
-//   return os;
-// }
+std::string ExprGroup::toString() const {
```

Comment:
Clang tidy used to complain when I left the debug printing uncommented, don't remember why. Are you not getting warnings while leaving this in?

Reply:
Apparently not.